gdcdictionary/schemas/treatment.yaml

```diff
@@ -504,6 +4627,126 @@ properties:
       - Not Allowed To Collect
     deprecated_enum:
       - Not Allowed To Collect
+    enumDef:
+      Other:
+        $ref:
+          - "_terms.yaml#/other/common"
+      Unknown:
+        $ref:
+          - "_terms.yaml#/unknown/common"
+      Not Reported:
+        $ref:
+          - "_terms.yaml#/not_reported/common"
+      Not Allowed To Collect:
+        $ref:
+          - "_terms.yaml#/not_allowed_to_collect/common"
+
+  treatment_arm:
+    $ref:
+      - "_terms.yaml#/treatment_arm/common"
```

Comment:
This PR is failing to build because `treatment_arm` is missing from the terms file.